lib/src/key/index.rs

```diff
@@ -29,6 +30,42 @@ impl<'a> Prefix<'a> {
 			tb,
 			_d: CHAR_INDEX,
```

Comment:
This is what I mean be shared definition. `use crate::key::CHAR_INDEX;`

Reply:
Thanks for pointing that out!